Python/ceval.c

```diff
@@ -5,7 +5,7 @@
 #include "Python.h"
 #include "pycore_abstract.h"      // _PyIndex_Check()
 #include "pycore_call.h"          // _PyObject_CallNoArgs()
-#include "pycore_ceval.h"         // _PyEval_SignalAsyncExc()
+#include "pycore_ceval.h"
```

Comment:
Is this change intentional? Otherwise, should probably revert it.

Reply:
It was intentional: I removed the `_PyEval_SignalAsyncExc()` function and replaced the one call to it (which wasn't even in this file) with `_Py_set_eval_breaker_bit(...)`. I tried removing the `#include` and saw that a number of other functions from `pycore_ceval.h` are still used.